db/clean_planting_csv/clean_planting_API_call.py

```diff
@@ -0,0 +1,97 @@
+import requests, zipfile, io
+import json
+import pandas as pd
+import uuid
+
+central_url = "https://senecaparkzoo.getodk.cloud"
+central_email = "email"
+central_password = "yourpassword"
+
+#gets authentication token
+def get_email_token():
+
+    email_token_response = requests.post(
+        central_url + "/v1/sessions",
+        data=json.dumps({"email": central_email, "password": central_password}),
+        headers={"Content-Type": "application/json"},
+    )
+
+    if email_token_response.status_code == 200:
+        return email_token_response.json()["token"]
+
+#gets json list of forms and form IDs - may be useful for future data imports
+#xmlFormId contains the form ID to put into form_response path for retrieving form submissions
+# def get_forms(email_token):
```

Comment:
Don't forget to delete commented out code that you don't need anymore

Reply:
For this block, I intended to leave it as a sign-post for the next person working with the ODK API. The commented-out function gives you a list of forms and form IDs, necessary for retrieving info from other forms. Should I un-comment this and just leave it as an un-called method in the script?

Reply:
Yeah I'd say uncomment it for now is fine.